### DISCUSSION TITLE
Auto-start Nebula on Android?

### DISCUSSION THREAD

**Discussion by liv2100** [slackhq/nebula#995] [Q&A]
Is there a method to auto-start the nebula service on an android device?  Other VPN clients can be setup in Android to auto-connect, but it seems Defined Networking Nebula VPN client doesn't support this - unless I'm missing something.

Has anyone found a way to have this vpn client auto-start on android?

**Comment by johnmaguire** [comment]
Hi @liv2100 - this is not currently possible, but please consider adding a vote using the 👍 emoji on https://github.com/DefinedNet/mobile_nebula/issues/49

If you have any further questions or feedback on the mobile app, please share them over on that repo. Thanks!
